Marketplace/Script_Combiner/AttachableScript.js

```diff
@@ -0,0 +1,221 @@
+(function() {
+
+    var _this = this;
+    //used to see if entities touched each other 
+    var entitiesTouched = true;
+    //used to see if entites are being held together
+    var entitiesTogether
+    //particle effect
+    var MAKING_CONNECTION_PARTICLE_PROPS;
+    //positions of both objects will be stored
+    var posA;
+    var posB;
+    //Sounds
+    var HANDSHAKE_SOUND_URL = "https://s3-us-west-1.amazonaws.com/hifi-content/davidkelly/production/audio/4beat_sweep.wav";
+    var SUCCESSFUL_HANDSHAKE_SOUND_URL = "https://s3-us-west-1.amazonaws.com/hifi-content/davidkelly/production/audio/3rdbeat_success_bell.wav";
+    //Variables to hold sounds
+    var handshakeInjector;
+    var successfulHandshakeInjector;
+    //particle effect holder
+    var particles //If this dosent work then remember to look back at code
+    
+    _this.preload = function(entityID) {
+        _this.entityID = entityID;
+        var props = Entities.getEntityProperties(entityID);
+        print("Loading attachable script properties");
+    }
+
+    function stopHandshakeSound() {
+        if (handshakeInjector) {
+            handshakeInjector.stop();
+            handshakeInjector = null;
+        }
+    }
+
+    _this.collisionWithEntity = function(idA, idB, collision) {
+        print("Entering entity!!!");
+        //returns a string of all userdata properties
+        var check1 = Entities.getEntityProperties(idB).userData;
+        print(Entities.getEntityProperties(idA).dynamic);
+        var isDynamic = 1;
+        //Checks to see if its a receiveable object and makes sure its not the stationary model with a dynamic check
+        if((JSON.parse(check1).canReceiveScripts == true) && (isDynamic == Entities.getEntityProperties(idA).dynamic) && (isDynamic == Entities.getEntityProperties(idB).dynamic)) {

```

Comment:
space after if and break this into multiple lines so it isn't over 128 characters long